docker/package/tezos_baking_wizard.py

```diff
@@ -345,7 +370,9 @@ def pprint_options(self):
     prompt="Provide the ledger URL for importing the baker secret key stored on a ledger.",
     help="Your ledger URL should look something like <ledger://<mnemonic>/ed25519/0'/1'>",
     default=None,
-    options=list_connected_ledgers(),
+    # We need a running node to gather all information about ledger-stored addresses balances
+    # so we're using 'None' here and update options later once the node is available
+    options=None,
```

Comment:
Good thinking :not-bad: Also keep in mind that the validator a couple lines below also uses `list_connected_ledgers()`, so you might want to re-assign the validator in the same place as `options`, too, once the node is up and running. Alternatively, you can move the entire step declaration to just before running that query, it'll probably be a bit cleaner even if it's not nice to separate step declarations.

Reply:
`list_connected_ledgers()` doesn't require tezos-node running, but probably it'll be better to run it only once to be sure that the list hasn't changed over time for some reason